playground/terraform/infrastructure/cloudbuild-manual-setup/README.md

```diff
@@ -0,0 +1,154 @@
+github_repository_branch      = "branch_name"               # The name of the GitHub repo branch to be connected with Cloud Build
+
+network_name                  = "network_name"              # GCP VPC Network Name for Playground deployment
+gke_name                      = "playground-backend"        # Playground GKE Cluster name
+state_bucket                  = "bucket_name"               # GCS bucket name for Beam Playground temp files
+```
+
+4. Create and fill `state.tfbackend` configuration file in same `environment_name` folder as follows:
+
+
+```
+bucket = "bucket_name"     # The name of bucket - will be used for terraform tfstate file
+```
+
+5. Push the `environment_name` folder to your GitHub repository `github_repository_name` and branch `github_repository_branch`.
+(Cloud Build needs file be present in GitHub repository it is connected to)
+
+**Note:** Some regions can be prohibited for Cloud Build. Please see [Cloud Build locations](https://cloud.google.com/build/docs/locations) for supported locations.
+
+6. Run commands:
+
+
+```console
+# Create a new authentication configuration for GCP Project with the created user account
+gcloud init
+
+# Command imports new user account credentials into Application Default Credentials
+gcloud auth application-default login
+
+# Navigate to 01.setup folder
+cd playground/terraform/infrastructure/cloudbuild-manual-setup/01.setup/
+
+# Run terraform scripts
+terraform init -backend-config="../../../environment/{environment_name}/state.tfbackend"
+terraform plan -var-file="../../../environment/{environment_name}/terraform.tfvars"
+terraform apply -var-file="../../../environment/{environment_name}/terraform.tfvars"
+```
+
+## 2. Connect GitHub repository and Cloud Build
+
+**Note:** Ensure correct `region` is set in [Cloud Build Triggers](https://console.cloud.google.com/cloud-build/triggers) page before proceeding further.
+
+Follow [Connect to a GitHub repository](https://cloud.google.com/build/docs/automating-builds/github/connect-repo-github)
+to connect the GitHub repository `https://github.com/{github_repository_owner}/{github_repository_name}` and `{github_branch_name}` with Cloud Build.
+
+## 3. Setup the Google Cloud Build triggers
+
+The `playground/terraform/infrastructure/cloudbuild-manual-setup/02.builders` provisions:
+- Cloud Build triggers to build and deploy Beam Playground.
+
+#### To execute the module
+
+
+```
+# Navigate to playground/terraform/infrastructure/cloudbuild-manual-setup/02.builders folder
+cd ../02.builders
+
+# Run terraform scripts
+terraform init -backend-config="../../../environment/{environment_name}/state.tfbackend"
+terraform plan -var-file="../../../environment/{environment_name}/terraform.tfvars"
+terraform apply -var-file="../../../environment/{environment_name}/terraform.tfvars"
+```
+
+## 4. Running the first Cloud Build trigger to deploy Playground infrastructure
```

Comment:
@ruslan-ikhsan Consider rephrasing, e.g.:
Run Cloud Build `Playground-infrastructure-trigger` to deploy Playground infrastructure